### DISCUSSION TITLE
Nightscout Device Authorization Failure

### DISCUSSION THREAD

**Discussion by mcrews1** [NightscoutFoundation/xDrip#2722] [Q&A]
RESOLVED: I am using Nightscout through Google Cloud. I am trying to deploy Nightscout but it is telling me my API Secret is incorrect. How do I fix this issue?  Thank you!!

**Comment by Navid200** [comment]
I can help.
Would you please post a screenshot of the status page?

**Reply by mcrews1** [reply]
As soon as I posted, I finally figured it out. I put resolved in the body. I have another issue with xDrip that I cannot figure out!!

**Reply by Navid200** [reply]
It would only help others if you explain what the resolution was.
Anyway, I have responded to your other post.
